target/linux/ramips/dts/mt7621_pangubox_pgb-m1.dts

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible =  "PanguBox,PGB-M1","mediatek,mt7621-soc";
```

Comment:
lower case

Reply:
And please tidy up spaces

Reply:
compatible = "pangubox,pgb-m1", "mediatek,mt7621-soc";

Reply:
compatible = "pangubox,pgb-m1", "mediatek,mt7621-soc";